flow/envs/base_env.py

```diff
@@ -103,11 +102,6 @@ def __init__(self, env_params, sim_params, scenario):
         # simulation step size
         self.sim_step = sim_params.sim_step
 
-        # the available_routes variable contains a dictionary of routes
-        # vehicles can traverse; to be used when routes need to be chosen
```

Comment:
where has this moved to?

Reply:
after the kernel instantiation (a few lines down)